empiric-package/empiric/core/client.py

```diff
@@ -38,7 +51,9 @@ async def get_decimals(self, key):
 
         return response.decimals
 
-    async def get_value(self, key, aggregation_mode, sources=None):
+    async def get_value(
+        self, key, aggregation_mode, sources=None
+    ) -> Tuple[Any, Any, Any, Any]:
```

Comment:
same here

Reply:
This will also be `int`s. Generally anything we get back from the network was a `felt` there so will be an `int` in Python. The one caveat is sometimes there are `string` return types if we convert the ascii-encoded felt to string within the function - not the case here though.